css-color-adjust-1/Overview.bs

```diff
@@ -606,13 +615,13 @@ Forced Color Palettes {#forced}
 				<tr>
 					<td>''Mark''</td>
 					<td>
-						N/A - this [=system color=] keyword should not be adjusted.
+						N/A - this <<system-color>> keyword should not be adjusted.
```

Comment:
The term "system color" exists, it's just not exported by accident. This (and other changes to `<<system-color>>` should be reverted; I'll go fix Colors to export the term "system color" properly.

Reply:
Ah ok good to know, I've reverted all those back to [=system color=].

Reply:
@tabatkins Interestingly, I still am seeing these not linking even though I see you landed a change to export "system color" from the color spec.

And if I run `bikeshed refs --text="system color"` all I get back is `[]`